nano/node/blockprocessor.cpp

```diff
@@ -382,7 +387,7 @@ void nano::block_processor::process_live (nano::block_hash const & hash_a, std::
 	});
 }
 
-nano::process_return nano::block_processor::process_one (nano::transaction const & transaction_a, nano::unchecked_info info_a)
+nano::process_return nano::block_processor::process_one (nano::transaction const & transaction_a, nano::unchecked_info info_a, const bool work_watcher_a)
```

Comment:
Shouldn't it also be called `watch_work_a` here?

Reply:
yup, thank you I thought I got them all but good catch